### PR TITLE
Fix Gherkin language execution parameters

### PR DESCRIPTION
**Pull request by brazilianbytes**
Cucumber 6.0.3 parameters are different from documentation.

### CODE REVIEW

File: content/docs/gherkin/reference.md [hunk 1]
```diff
@@ -442,22 +442,29 @@ In order to allow Gherkin to be written in a number of languages, the keywords h
 
 ### List translation options
 You can get information about the translations from the command line.
+Note: 'cucumber' command may be different over code languages. ex. for Node.js is 'cucumber-js'
 
 To see a listing of available languages:
 ```shell
-cucumber --i18n help
+cucumber --i18n-languages help
```
Comment (aslakhellesoy) [inline content/docs/gherkin/reference.md:449]: Unfortunately, this option is named differently across platforms, so changing from `--i18n` to `--i18n-languages` makes this incorrect for cucumber-ruby and cucumber-jvm.

I think we should simply get rid of the option and refer users to the docs.

Comment (aslakhellesoy) [reply content/docs/gherkin/reference.md:449]: See https://github.com/cucumber/cucumber/issues/783

**Comment by aslakhellesoy** [comment]
Should we just close this, or do we want to make the docs language specific? WDYT @mlvandijk ?

**Comment by mlvandijk** [comment]
@aslakhellesoy We don't currently have different language tabs on the Gherkin page(s) as Gherkin should be programming language agnostic.

We could just remove this section imho. - what do you think?

**Comment by mlvandijk** [comment]
Thanks for pointing out the inconsistency though @brazilianbytes!

**Comment by aslakhellesoy** [comment]
There are two suggestions here:

1) Remove the docs about `--i18n`
2) Remove `--i18n` from Cucumbers - see https://github.com/cucumber/cucumber/issues/783

I think we should remove both.

**Comment by mlvandijk** [comment]
@aslakhellesoy Can we point them anywhere to see up to date available languages (rather than duplicate in documentation?). Kind of like we do for DataTables (points to datatable README from the docs)

**Comment by aslakhellesoy** [comment]
Yes @mlvandijk here: https://cucumber.io/docs/gherkin/reference/#gherkin-dialects

**Comment by mlvandijk** [comment]
Perfect! Let's link there. @brazilianbytes - would you be willing to do this change instead?

**Comment by mlvandijk** [comment]
Closed in favor of cucumber/common#431

**Pull request closed without merging**